Source/Urho3D/Resource/Decompress.cpp

```diff
@@ -269,187 +278,7 @@ void DecompressImageDXT(unsigned char* rgba, const void* blocks, int width, int
-        blue2 = (unsigned char)((blockTop & 0xf0000) >> 12);
-        blue2 = blue2 + (blue2 >> 4);    // copy bits to lower sig
-        green2 = (unsigned char)((blockTop & 0xf00) >> 4);
-        green2 = green2 + (green2 >> 4);    // copy bits to lower sig
-        red2 = (unsigned char)((blockTop & 0xf) << 4);
-        red2 = red2 + (red2 >> 4);    // copy bits to lower sig
-    }
-    // get the modtables for each subblock
-    modtable1 = (int)((blockTop >> 29) & 0x7);
-    modtable2 = (int)((blockTop >> 26) & 0x7);
-
-    if (!bFlip)
-    {   // 2 2x4 blocks side by side
-        for (int j = 0; j < 4; j++)    // vertical
-        {
-            for (int k = 0; k < 2; k++)    // horizontal
-            {
-                *(output + j * 4 + k) = ModifyPixel(red1, green1, blue1, k, j, blockBot, modtable1);
-                *(output + j * 4 + k + 2) = ModifyPixel(red2, green2, blue2, k + 2, j, blockBot, modtable2);
-            }
-        }
-    }
-    else
-    {   // 2 4x2 blocks on top of each other
-        for (int j = 0; j < 2; j++)
-        {
-            for (int k = 0; k < 4; k++)
-            {
-                *(output + j * 4 + k) = ModifyPixel(red1, green1, blue1, k, j, blockBot, modtable1);
-                *(output + (j + 2) * 4 + k) = ModifyPixel(red2, green2, blue2, k, j + 2, blockBot, modtable2);
-            }
-        }
-    }
-}
-
-void DecompressImageETC(unsigned char* rgba, const void* blocks, int width, int height)
-{
-    // initialise the block input
-    auto const* sourceBlock = reinterpret_cast< unsigned char const* >( blocks );
-    int bytesPerBlock = 8;
-
-    // loop over blocks
-    for (int y = 0; y < height; y += 4)
-    {
-        for (int x = 0; x < width; x += 4)
-        {
-            // decompress the block
-            unsigned char targetRgba[4 * 16];
-            DecompressETC(targetRgba, sourceBlock);
-
-            // write the decompressed pixels to the correct image locations
-            unsigned char const* sourcePixel = targetRgba;
-            for (int py = 0; py < 4; ++py)
-            {
-                for (int px = 0; px < 4; ++px)
-                {
-                    // get the target location
-                    int sx = x + px;
-                    int sy = y + py;
-                    if (sx < width && sy < height)
-                    {
-                        unsigned char* targetPixel = rgba + 4 * (width * sy + sx);
-
-                        // copy the rgba value
-                        for (int i = 0; i < 4; ++i)
-                            *targetPixel++ = *sourcePixel++;
-                    }
-                    else
-                    {
-                        // skip this pixel as its outside the image
-                        sourcePixel += 4;
-                    }
-                }
-            }
-
-            // advance
-            sourceBlock += bytesPerBlock;
-        }
-    }
-}
+// PVRTC decompression based on the Oolong Engine, modified for Urho3D

```

Comment:
Is this comment relevant?

Reply:
It just changed comment
`// ETC and PVRTC decompression based on the Oolong Engine, modified for Urho3D`
To
`// PVRTC decompression based on the Oolong Engine, modified for Urho3D`